src/devices/slacky_diy.ts

```diff
@@ -1917,4 +2016,168 @@ export const definitions: DefinitionWithExtend[] = [
         meta: {},
         ota: true,
     },
+    {
+        zigbeeModel: ["AirQ_Monitor_S01"],
+        model: "AirQ_Monitor_S01",
+        vendor: "Slacky-DIY",
+        description: "Air quality monitor",
+        configure: async (device, coordinatorEndpoint, logger) => {
+            const endpoint = device.getEndpoint(1);
+            await endpoint.read("genLevelCtrl", ["currentLevel"]);
+        },
+        extend: [
+            m.co2({reporting: {min: 10, max: 3600, change: 0.00001}}),
+            m.numeric({
+                name: "voc_index",
+                access: "STATE_GET",
+                cluster: "genAnalogInput",
+                attribute: "presentValue",
+                reporting: {min: 10, max: 3600, change: 30},
+                unit: "VOC Index points",
+                description: "VOC index",
+            }),
+            m.temperature({reporting: {min: 10, max: 3600, change: 10}}),
+            m.humidity({reporting: {min: 10, max: 3600, change: 10}}),
```

Comment:
```suggestion
            m.humidity(),
```

Please use the default reporting values (same for `temperature`)